File: fibonacci.java
/*
Enunciado: Escribir una función en Java que devuelva el n-ésimo término de la secuencia de Fibonacci.
*/

public class Fibonacci {

    public static int fibonacci(int n) {
        if (n <= 1)
            return n;
        else
            return fibonacci(n - 1) + fibonacci(n - 2);
    }

    public static void main(String[] args) {
        int n = 10; // Número de términos da secuencia de Fibonacci a calcular
        System.out.println("O " + n + "º término da secuencia de Fibonacci é: " + fibonacci(n));
    }
}

File: fibonacci.py
'''
Enunciado: Escribir una función en Python que devuelva el n-ésimo término de la secuencia de Fibonacci.
'''

def fibonacci(n):
    if n <= 1:
        return n
    else:
        return fibonacci(n - 1) + fibonacci(n - 2)

if __name__ == "__main__":
    n = 10  # Número de términos da secuencia de Fibonacci a calcular
    print(f"O {n}º término da secuencia de Fibonacci é: {fibonacci(n)}")
